tests/baselines/reference/tsserver/fourslashServer/autoImportProvider_importsMap1.js

```diff
@@ -0,0 +1,357 @@
+currentDirectory:: / useCaseSensitiveFileNames: false
```

Comment:
What’s generating these baselines?

Reply:
New in #56132.

Reply:
hah, I'm not sure :D when I was developing this I was just running `hereby runtests --tests=autoImportProvider_importsMap1 --no-lint` and it didn't generate them (nor it was complaining about lack of them). However, when I executed `hereby runtests-parallel --no-lint` at the end of the development to confirm that I didn't break anything else... those appeared (well, the test run failed and I had to accept those)